dist/docker/redhat/build_docker.sh

```diff
@@ -89,7 +98,7 @@ bcp LICENSE-ScyllaDB-Source-Available.md /licenses/
 run microdnf clean all
 run microdnf --setopt=tsflags=nodocs -y update
 run microdnf --setopt=tsflags=nodocs -y install hostname kmod procps-ng python3 python3-pip
-run microdnf clean all
+run curl -L --output /etc/yum.repos.d/scylla.repo ${repo_file_url}
```

Comment:
Q: will repo_file_url be available even before building the first release candidate of a branch?

If not, we'll get into an endless loop. Need to make sure repo_file_url is valid before building anything on a branch.

Reply:
it will be created once we spin rc0, and this is happening during branching process